(dp1
S'output'
p2
S'<class \'gluon.contrib.pymysql.err.InternalError\'> (1553, u"Cannot drop index \'autor__idx\': needed in a foreign key constraint")'
p3
sS'layer'
p4
S'/home/janderson/PycharmProjects/untitled1/web2py/applications/Escola/models/table.py'
p5
sS'code'
p6
S'# -*- coding: utf-8 -*-\n__author__ = \'janderson\'\n\nfrom datetime import datetime\n\n#Cria\xc3\xa7\xc3\xa3o tabelas notas\nNotas = db.define_table(\'notas\',\n    Field(\'nota\' , \'float\' ,default=0 , label="Nota" ),\n    Field(\'aluno\' , \'reference auth_user\' , notnull=True , label="Aluno"),\n    Field(\'professor\' , \'reference auth_user\' , notnull=True , label="Professor")\n)\n\n#Cria\xc3\xa7\xc3\xa3o tabela biblioteca\nBiblioteca = db.define_table(\'biblioteca\',\n    Field(\'arquivo\' , \'upload\' , notnull=True , label="Arquivo"),\n    Field(\'professor\' , \'reference auth_user\' , notnull=True , label="Professor")\n)\n\n#Cria\xc3\xa7\xc3\xa3o tabela Forum\nForum = db.define_table(\'forum\',\n    Field(\'Titulo\', notnull=True , label="Titulo"),\n    Field(\'mensagem\' , \'text\' , notnull=True , label="Mensagem"),\n    auth.signature\n)\n\n#Cria\xc3\xa7\xc3\xa3o tabela Comentarios\nComentarios = db.define_table(\'comentarios\',\n    Field(\'mensagem\' , \'text\' , notnull=True , label="Mensagem"),\n    Field(\'postagem\' , \'reference forum\' , label="Postagem"),\n    auth.signature\n)\n\n\n\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/janderson/PycharmProjects/untitled1/web2py/gluon/restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "/home/janderson/PycharmProjects/untitled1/web2py/applications/Escola/models/table.py", line 23, in <module>\n    auth.signature\n  File "/home/janderson/PycharmProjects/untitled1/web2py/gluon/packages/dal/pydal/base.py", line 819, in define_table\n    table = self.lazy_define_table(tablename,*fields,**args)\n  File "/home/janderson/PycharmProjects/untitled1/web2py/gluon/packages/dal/pydal/base.py", line 858, in lazy_define_table\n    polymodel=polymodel)\n  File "/home/janderson/PycharmProjects/untitled1/web2py/gluon/packages/dal/pydal/adapters/base.py", line 503, in create_table\n    fake_migrate=fake_migrate\n  File "/home/janderson/PycharmProjects/untitled1/web2py/gluon/packages/dal/pydal/adapters/base.py", line 616, in migrate_table\n    self.execute(sub_query)\n  File "/home/janderson/PycharmProjects/untitled1/web2py/gluon/packages/dal/pydal/adapters/base.py", line 1378, in execute\n    return self.log_execute(*a, **b)\n  File "/home/janderson/PycharmProjects/untitled1/web2py/gluon/packages/dal/pydal/adapters/base.py", line 1372, in log_execute\n    ret = self.cursor.execute(command, *a[1:], **b)\n  File "/home/janderson/PycharmProjects/untitled1/web2py/gluon/contrib/pymysql/cursors.py", line 117, in execute\n    self.errorhandler(self, exc, value)\n  File "/home/janderson/PycharmProjects/untitled1/web2py/gluon/contrib/pymysql/connections.py", line 202, in defaulterrorhandler\n    raise errorclass, errorvalue\nInternalError: (1553, u"Cannot drop index \'autor__idx\': needed in a foreign key constraint")\n'
p11
s.